File: src/content/docs/usage-examples/color/alpha_of/alpha_of-1-red-50pct-transparency.cs
using SplashKitSDK;

// Define a color with RGBA values
Color myColor = SplashKit.RGBAColor(255, 0, 0, 128); // Red with 50% transparency

// Get the alpha value of the color
int alphaValue = SplashKit.AlphaOf(myColor);

// Print the alpha value
SplashKit.WriteLine("The alpha value of myColor is: " + alphaValue);

// Open a window
Window window = SplashKit.OpenWindow("AlphaOf Example", 400, 400);

// Draw a filled circle on the screen using the color from above
SplashKit.ClearScreen();
SplashKit.FillCircle(myColor, 200, 200, 100);
SplashKit.RefreshScreen();

// Keep the window open for 5 seconds
SplashKit.Delay(5000);

// Close the window
SplashKit.CloseWindow(window);

File: src/assets/usage-examples-code/color/alpha_of/alpha_of-1-red-50pct-transparency.cpp
#include "splashkit.h"

using std::to_string;

int main()
{
    // Define a color with an alpha value (Red with 50% transparency)
    color my_color = rgba_color(255, 100, 50, 255);

    // Get the alpha value of the color
    int alpha = alpha_of(my_color);

    // Print the alpha value
    write_line("The alpha value of my_color is: " + to_string(alpha));

    // Open a window
    open_window("Alpha Of Example", 400, 400);

    // Draw a filled circle on the screen using the color from above
    clear_screen();
    fill_circle(my_color, 200, 200, 100);
    refresh_screen();

    // Keep the window open for 5 seconds
    delay(5000);

    // Close the window
    close_window("Alpha Of Example");
    

    return 0;
}
